### PR TITLE
VIT-1191 Update docs for freestyle multi-region support

### PR DESCRIPTION
**Pull request by ib250**

### CODE REVIEW

**Review by gabrieleangeletti — COMMENTED** [review]
Pardon my ignorance, but why are we deleting `swagger.json`?

**Comment by ib250** [comment]
> Pardon my ignorance, but why are we deleting `swagger.json`?

😮 it should be a diff not a delete, I'll check this. For some reason the deployment is working fine so I'm not sure what is going on...

File: docs/providers/Guides/Freestyle.mdx [hunk 1]
```diff
@@ -4,7 +4,8 @@ import { CodeBlock } from "@theme/codeblock";
 # Freestyle
 
 :::note
-Please make sure you invite Vital to your practice as a clinician with admin rights. To connect a user to Vital you must first connect your Freestyle practice to Vital.
+Please make sure you invite Vital to your practice as a clinician with admin rights. To connect a user to Vital you must first connect your Freestyle practice to Vital in the appropriate Vital region for your Freestyle account.
+
```
[the remark this answers is not in the document]
Comment (gabrieleangeletti) [reply docs/providers/Guides/Freestyle.mdx:8]: Yeah let's go with link it

**Comment by gabrieleangeletti** [comment]
lolololol

**Review by gabrieleangeletti — APPROVED** [review]

**Pull request merged**